Respnose caching enhancements (#15489)
aspnet/AspNetCore.Docs: aspnetcore/performance/caching/middleware.md

::: moniker range=">= aspnetcore-3.0"

> Response Caching Middleware is made available by the [Microsoft.AspNetCore.ResponseCaching](https://www.nuget.org/packages/Microsoft.AspNetCore.ResponseCaching/) package, which is implicitly added to ASP.NET Core apps.

Reviewer: I'd rephrase this in the 3.0+ moniker range. The package doesn't exist anymore (notice there's no 3.0+ versions), it's part of the `AspNetCore.App` framework. Maybe just:

```suggestion
The Response Caching middleware is available in all ASP.NET Core apps.
```
